spec/support/comment.cr

```diff
@@ -6,6 +6,8 @@ class Comment < BaseModel
     timestamps
     column body : String
     belongs_to post : Post
+    polymorphic_belongs_to commentable : Employee | Company
+    polymorphic_belongs_to commentable_opt : Employee | Company | Nil
```

Comment:
Could you call this column `optional_commentable`? I kept thinking this was `otp` like a one time password